### PR TITLE
Feature lwdev 5200 add fee data to output

### PR DESCRIPTION
**Pull request by tarurar**

### CODE REVIEW

File: src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs [hunk 1]
```diff
@@ -51,5 +52,21 @@ public string VolumeText
         }
 
         public string PreviousId { get; set; }
+
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs:57]: Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.

File: src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/ClientTradesRepository.cs [hunk 2]
```diff
@@ -47,6 +48,22 @@ public TransactionStates State
         public double Volume { get; set; }
         public string ClientId { get; set; }
 
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
+        public string FeeTypeText
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/ClientTradesRepository.cs:53]: Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.

File: src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/TransferEventsRepository.cs [hunk 3]
```diff
@@ -39,6 +40,22 @@ public TransactionStates State
         }
         public string ClientId { get; set; }
 
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
+        public string FeeTypeText
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/TransferEventsRepository.cs:45]: Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.

File: client/Lykke.Service.OperationsCache.Client/AutorestClient/Models/HistoryEntry.cs [hunk 4]
```diff
@@ -22,7 +22,7 @@ public HistoryEntry()
         /// <summary>
         /// Initializes a new instance of the HistoryEntry class.
         /// </summary>
-        public HistoryEntry(string id = default(string), System.DateTime? dateTime = default(System.DateTime?), double? amount = default(double?), string currency = default(string), string clientId = default(string), string customData = default(string), string opType = default(string))
+        public HistoryEntry(double feeSize, string id = default(string), System.DateTime? dateTime = default(System.DateTime?), double? amount = default(double?), string currency = default(string), string clientId = default(string), string customData = default(string), string opType = default(string), string feeType = default(string))
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.OperationsCache.Client/AutorestClient/Models/HistoryEntry.cs:25]: Use a C# 7.1 feature  - default without type definition
Ah. It is auto generated code. Nothing need to do here

File: client/Lykke.Service.OperationsCache.Client/Models/HistoryClientEntry.cs [hunk 5]
```diff
@@ -11,6 +11,8 @@ public class HistoryClientEntry
         public string ClientId { get; set; }
         public string CustomData { get; set; }
         public string OpType { get; set; }
+        public double FeeSize { get; set; }
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.OperationsCache.Client/Models/HistoryClientEntry.cs]: Add a description for a property in a shared model

File: src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs (hunk 1 above)
Comment (tarurar) [reply src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs:57]: Why? FeeType can be Absolute or Relative. This value can't be inferred from the value of FeeSize or vice versa.

Comment (MikhailGerasimov) [reply src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs:57]: My comment was about FeeTypeText. Sorry for the confusion.

Comment (tarurar) [reply src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs:57]: You are right, removed setters.

**Review by MikhailGerasimov — APPROVED** [review]

**Pull request merged**